src/Integration.Vsix.UnitTests/SonarLintTagger/TextBufferIssueTrackerTests.cs

```diff
@@ -204,7 +196,7 @@ public void WhenNewIssuesAreFound_ButForIncorrectFile_ListenersAreNotUpdated()
             var errorListSinkMock2 = RegisterNewErrorListSink();
 
             var testSubject = new TextBufferIssueTracker(taggerProvider.dte, taggerProvider,
-                mockedJavascriptDocumentFooJs.Object, javascriptLanguage, new TestLogger());
+                mockedJavascriptDocumentFooJs.Object, javascriptLanguage, new TestLogger(), issuesFilter.Object);
```

Comment:
For me, creating a `testSubject` in `TestInitialize` then overriding it in a test method is a poor practice; it's inconsistent and clumsy. Personally, I'd create a separate `CreateDefaultTestSubject` method and explicitly call it from the tests that use it.